scripts/lsf/run.sh

```diff
@@ -97,7 +130,7 @@ do
         
         savehosts=${savehosts%?}
         thehost='\`hostname\`'
-        echo "pdsh -w $USER@$savehosts ${REDISHOME}/bin/redis-cli -h %h -p $PORT -a $PW BGSAVE" >>redis-save-rdb.sh
+        echo "pdsh -w $USER@$savehosts $WORKDIR/saverdb.sh %h $REDISHOME $PW $PORT" >>redis-save-rdb.sh
```

Comment:
usage of WORKDIR and REDISHOME might fail if there are any special chars in those paths. Suggesting to wrap them in {}

Reply:
oh, on line 133? Yes, I will modify

Reply:
updated